auth/authRoute.js

```diff
@@ -37,25 +37,28 @@ const Users = require("../models/admin_user");
 router.post("/login", (req, res) => {
   const { email, password } = req.body;
   if (email && password) {
-    Users.findByEmail(email)
-      .then(user => {
-        if (user && bcrypt.compareSync(password, user.password)) {
-          const token = generateToken(user);
-          res.status(200).json({
-            user: { ...user, password: "noneya:)" },
-            token: token
-          });
-        } else {
-          res.status(400).json({
-            message: "Invalid credentials."
-          });
-        }
-      })
-      .catch(err =>
-        res.status(500).json({
-          message: "We couldn't process your login at the moment"
-        })
-      );
+      if (typeof(email) === "string" && typeof(password) === "string") {
+		    Users.findByEmail(email)
+		      .then(user => {
+				if (user && bcrypt.compareSync(password, user.password)) {
+				  delete user.password;
+				  const token = generateToken(user);
+				  res.status(200).json({
+					user: { ...user },
+					token: token
+				  });
+				} else {
+			  		res.status(400).json({
+			    		message: "Invalid credentials."
+			  		});
+				}
+		      })
+		      .catch(err =>
+				res.status(500).json({
+			  		message: "We couldn't process your login at the moment"
+				})
+		      );
+	  }
```

Comment:
@zoverlvx We should probably add an else statement that covers fallback when password/email aren't strings